.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- 撰写您想附加的信息 -->
+<!-- Write your own things -->
+
+**Please check your copyright declearation**
+
+- [ ] By making this pull request, I declare that I legally own the copyright and other related rights of the plugin which I submitted, and that any adverse consequences arising from a false declaration shall be borne by me. 通过提交此拉取请求，我声明我合法拥有我提交的插件的著作权和其它有关权利，由虚假声明产生的不利后果由我本人承担。

```

Comment:
Why should it be so formal. I don't like to require people to declare stufs like this just for a simple plugin request